assets/stylesheets/application.scss

```diff
@@ -35,22 +35,22 @@ $text-danger-color: #fff;
 // ----------------------------------------------------------------------------
 // Base styles
 // ----------------------------------------------------------------------------
-html { 
-  font-size: 100%; 
-  -webkit-text-size-adjust: 100%; 
-  -ms-text-size-adjust: 100%; 
+html {
+  font-size: 100%;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
 }
 
-body { 
-  margin: 0; 
+body {
+  margin: 0;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, margin